### ISSUE TITLE
todo/onboarding issues

### ISSUE THREAD

**Issue by OAyomide** [darmie/zflow#1]
Are there issues that could be taken a look into, to help ease into development. useful to get started to the codebase and know where to get in. Perhaps CLI/chore issues.

**Comment by darmie** [comment]
Yes please, I'll definitely need help with documentation. My focus is to get the Graph network to work first, then we can have a proper documentation for implementers.
